apps/emqx/rebar.config

```diff
@@ -24,7 +24,7 @@
 {deps, [
     {emqx_utils, {path, "../emqx_utils"}},
     {lc, {git, "https://github.com/emqx/lc.git", {tag, "0.3.2"}}},
-    {gproc, {git, "https://github.com/emqx/gproc", {tag, "0.9.0.1"}}},
+    {gproc, {git, "https://github.com/uwiger/gproc", {tag, "0.9.1"}}},
```

Comment:
better to sync and use our fork.

Reply:
Our `0.9.0.1` was tagged just because `0.9.1` was not ready upstream yet.  Both are basically at the same state, just `0.9.0.1` has appup for v4.4.

I talked with @zmstone , and I understood that we wanted to use upstream if possible here. :thinking:

Reply:
Sorry.
For gproc, since we already forked (with appup etc in place), it makes more sense to use our fork instead.